modules/df/df_tools/df_tools_blocks/css/hero.css

```diff
@@ -0,0 +1,59 @@
+.hero-block .full-width-inner {
+  position: relative;
+}
+
+.hero-block .hero-block-fields {
+  position: absolute;
+  padding: 20px;
+  margin-top: 10%;
+  width: 100%;
+}
+
+.hero-block .hero-block-fields.hero-block-align-right {
+  text-align: right;
+}
+
+.hero-block .hero-block-fields.hero-block-align-left {
+  text-align: left;
+}
+
+.hero-block .hero-block-fields.hero-block-align-center {
+  text-align: center;
+}
+
+.hero-block .field-name-field-first-line {
+  font-size: 2rem;
+}
+
+.hero-block .field-name-field-second-line {
+  font-size: 1rem;
+  margin-bottom: .5rem;
+}
+
+.hero-block .field-name-field-hero-link a {
+  color: white;
+  border: 1px solid white;
+  background: #2199e8;
+  padding: 10px;
+  filter: brightness(100%);
+  transition: .2s;
+}
+
+.hero-block .field-name-field-hero-link a:hover {
+  filter: brightness(85%);
```

Comment:
This is a creative way to generically apply hover styles to colors without manually writing in rules. This way the user can set the background to anything, and we calculate a darker hover color for it.